website/docs/feature-flag-tutorials/react/examples.md

```diff
@@ -0,0 +1,627 @@
+```
+
+Next, within the `NavBar` component itself, call `useUnleashClient` and wrap a `useEffect` hook around our function calls to grab impression data with this code snippet:
+
+```js
+ const unleashClient = useUnleashClient();
+
+ useEffect(() => {
+   unleashClient.start();
+
+   unleashClient.on("ready", () => {
+     const enabledImpression = unleashClient.isEnabled("newNotificationsBadge");
+     console.log(enabledImpression);
+   });
+
+   unleashClient.on("impression", (impressionEvent: object) => {
+     console.log(impressionEvent);
+     // Capture the event here and pass it internal data lake or analytics provider
+   });
+ }, [unleashClient]);
+```
+
+This code snippet starts the Unleash client, checks that our flag is enabled, and then stores impression events for your use.
+
+> **Note:** We are passing in unleashClient into the dependency array in useEffect to prevent the app from [unnecessarily mounting the component](https://react.dev/reference/react/useEffect#parameters) if the state of the data it holds has not changed.
+
+Our flag impression data is now being logged!
+
+In your browser, you can view the output for `isEnabled`:
+
+```js
+{
+    "eventType": "isEnabled",
+    "eventId": "b4455218-4a88-43aa-8712-b99186f46548",
+    "context": {
+        "sessionId": 386689528,
+        "appName": "cypress-realworld-app",
+        "environment": "default"
+    },
+    "enabled": true,
+    "featureName": "newNotificationsBadge",
+    "impressionData": true
+}
+```
+
+And the console.log for `getVariant` returns:
+
+```js
+{
+    "eventType": "getVariant",
+    "eventId": "c41aa58b-d2c7-45cf-b668-7267f465e01a",
+    "context": {
+        "sessionId": 386689528,
+        "appName": "cypress-realworld-app",
+        "environment": "default"
+    },
+    "enabled": true,
+    "featureName": "newNotificationsBadge",
+    "impressionData": true,
+    "variant": "announcementsIcon"
+}
+```
+
+You can find more information on `isEnabled` and `getVariant` in our [impression data docs](/reference/impression-data#impression-event-data).
+
+Now that the application is capturing impression events, you can configure the correct data fields and formatting to send to any analytics tool or data warehouse you use.
+
+
+### Application Metrics & Monitoring
+
+Under the [Metrics tab](http://localhost:4242/projects/default/features/newNotificationsBadge/metrics), you can see the general activity of the [Cypress Real World App from our React tutorial](/feature-flag-tutorials/react) in the development environment over different periods of time. If the app had a production environment enabled, we would also be able to view the amount of exposure and requests the app is receiving over time.
+
+
+![Metrics view image](/img/react-ex-metrics.png)
+
+
+Our metrics are great for understanding user traffic. You can get a better sense of:
+
+
+* What time(s) of the day or week are requests the highest?
+* Which feature flags are the most popular?
```

Comment:
Hmm, are feature flags popular? Maybe it'd be better worded as "which feature flags receive the most traffic" or something?

```suggestion
* Which feature flags receive the most traffic?
```